### PR TITLE
wg-k8s-infra: Add presubmit to validate k8s resources

### PR DESCRIPTION
**Pull request by ameukam**
Ref: https://github.com/kubernetes/k8s.io/issues/1734

/wg k8s-infra

Signed-off-by: Arnaud Meukam <ameukam@gmail.com>

### CODE REVIEW

File: config/jobs/kubernetes/wg-k8s-infra/k8sio-presubmit.yaml [hunk 1]
```diff
@@ -33,3 +33,18 @@ presubmits:
...
+      testgrid-tab-name: k8sio-k8s-resources-verify
+    decorate: true
+    path_alias: k8s.io/k8s.io
+    run_if_changed: '\.yaml$'
+    spec:
+      containers:
+      - image: openpolicyagent/conftest:latest
+        args:
+        - test
+        - -p policies/ */*.yaml
```
Comment (spiffxp) [inline config/jobs/kubernetes/wg-k8s-infra/k8sio-presubmit.yaml:50]: Is this one level deep or recursive descent?

File: config/jobs/kubernetes/wg-k8s-infra/k8sio-presubmit.yaml [hunk 2]
```diff
@@ -33,3 +33,18 @@ presubmits:
...
         - ./hack/verify.sh
+  - name: k8sio-k8s-resources-verify
+    annotations:
+      testgrid-dashboards: wg-k8s-infra-k8sio
+      testgrid-num-failures-to-alert: '1'
+      testgrid-alert-email: ameukam@gmail.com
+      testgrid-tab-name: k8sio-k8s-resources-verify
+    decorate: true
+    path_alias: k8s.io/k8s.io
+    run_if_changed: '\.yaml$'
```
Comment (spiffxp) [inline config/jobs/kubernetes/wg-k8s-infra/k8sio-presubmit.yaml:44]: Can we isolate this to paths with k8s yaml? There is some yaml that is not k8s

**Review by spiffxp — CHANGES_REQUESTED** [review]

**Comment by ameukam** [comment]
Need to update this with the recent changes in https://git.k8s.io/k8s.io

/remove-lifecycle rotten

**Pull request closed without merging**